### PR TITLE
[WIP] Add export feature

### PR DESCRIPTION
**Pull request by semtexzv**
- Added endpoints for exporting data
- Endpoints do not support paging, sorting, or filters(TBD)
- Supporting 'application/json' & 'text/csv' content types
- Flat object structure for easy manipulation

### CODE REVIEW

File: manager/controllers/systems.go [hunk 1]
```diff
@@ -93,14 +97,60 @@ func SystemsListHandler(c *gin.Context) {
...
-	var resp = SystemsResponse{
+	resp := SystemsResponse{
 		Data:  data,
 		Links: *links,
 		Meta:  *meta,
 	}
 	c.JSON(http.StatusOK, &resp)
 }
 
+// nolint: gocritic, lll
```
Comment (Josca) [inline manager/controllers/systems.go:108]: same question for lint as upper

File: manager/controllers/systems_test.go [hunk 2]
```diff
@@ -117,3 +118,40 @@ func TestSystemsSearch(t *testing.T) {
...
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "/", nil)
+	req.Header.Add("Accept", "application/json")
+	core.InitRouterWithPath(SystemsExportHandler, "/").ServeHTTP(w, req)
+
+	assert.Equal(t, 200, w.Code)
+	var output []SystemInlineItem
+
+	ParseReponseBody(t, w.Body.Bytes(), &output)
```
[the remark this answers is not in the document]
Comment (semtexzv) [reply manager/controllers/systems_test.go:134]: Well, we use the same approach for testing other endpoints

File: manager/controllers/systems.go [hunk 3]
```diff
@@ -93,14 +97,60 @@ func SystemsListHandler(c *gin.Context) {
...
+	err := query.Find(&systems).Error
+	if err != nil {
+		LogAndRespError(c, err, "db error")
+		return
+	}
+
+	data := make([]SystemInlineItem, len(systems))
+
+	for i, v := range systems {
+		data[i] = SystemInlineItem(v)
```
[the remark this answers is not in the document]
Comment (semtexzv) [reply manager/controllers/systems.go:133]: Vulnerability uses the inline variant everywhere.

File: docs/openapi.json [hunk 4]
```diff
@@ -714,6 +784,32 @@
                     }
                 }
             },
+            "controllers.AdvisoryInlineItem": {
```
[the remark this answers is not in the document]
Comment (semtexzv) [reply docs/openapi.json:787]: Well, it's not optimal, but If we want to have sane export format and preserve API format, we have to do it this way. In the code both of these use shared structure for denoting attributes, and API is generated automatically, so it's not a problem to modify attributes down the line.

**Pull request merged**